doc/api/child_process.md

```diff
@@ -299,6 +299,9 @@ output on this fd is expected to be line delimited JSON objects.
 *Note: Unlike the fork(2) POSIX system call, `child_process.fork()` does
 not clone the current process.*
 
+*Note*: Support for non-Array values of the `.stdio` option was added in
```

Comment:
AFAIK we haven't been including version information when it comes to changes in behavior over time. That might be beneficial, but still it's a separate discussion on *if* we want to start doing that and if so, *how* something like that would be best conveyed to the end user (presumably not in a free-form way like this note).